lua/pckr/loader.lua

```diff
@@ -214,7 +214,9 @@ local function load_plugins()
   -- Load pckr plugins
   for _, plugin in pairs(plugins) do
     if not plugin.cond then
-      M.load_plugin(plugin)
+      if plugin.added then
```

Comment:
This looks wrong.

If a plugin is required by another, it will never be loaded.

Reply:
In `function M.load_plugin(plugin)`, a plugin is required by another will be loaded:
```lua
  if plugin.requires then
    log.fmt_debug('Loading dependencies of %s', plugin.name)
    local all_plugins = require('pckr.plugin').plugins

    for _, name in ipairs(plugin.requires) do
      M.load_plugin(all_plugins[name])
    end
  end
```